dkms.in

```diff
@@ -916,11 +916,19 @@ prepare_signing()
     fi
     echo "Public certificate (MOK): $mok_certificate"
 
-    if ( [ ! -f $mok_signing_key ] && [[ ! "$mok_signing_key" == *":"* ]] ) || [ ! -f $mok_certificate ]; then
+    if [[ ! "$mok_signing_key" == "pkcs11:"* ]] && ([ ! -f "$mok_signing_key" ] || [ ! -f "$mok_certificate" ]); then
```

Comment:
Why the "pkcs11:" check - should we update the documentation for `mok_signing_key` saying it must include that?

Reply:
In addition to taking a filename as the signing key, sign-file also supports using pkcs11:*
https://github.com/torvalds/linux/blob/bb1a1146467ad812bb65440696df0782e2bc63c8/scripts/sign-file.c#L142-L175
And the issue https://github.com/dell/dkms/issues/240

Reply:
That kernel code has been around for 7 years ouch. Seems like the manual page should have been updated alongside the code.